torch/fx/graph.py

```diff
@@ -648,8 +648,31 @@ def emit_node(node: Node):
                     "val",
                     node.meta.get("tensor_meta", node.meta.get("example_value", None)),
                 )
+                # Tensor subclass printing support
+                if (
+                    issubclass(meta_val.__class__, torch.Tensor)
```

Comment:
This is still not very idiomatic. More idiomatic would be `isinstance(meta_val, torch.Tensor) and type(meta_val) is not torch.Tensor`